CONTRIBUTING.md

```diff
@@ -52,9 +52,5 @@ Once the necessary changes are made, run the following to generate the python co
 
 ```
 nbdev_build_lib
```

Comment:
hi @joshuacortez it should be `nbdev_export`. 
`nbdev_build_lib` is no [longer supported](https://nbdev.fast.ai/api/doclinks.html#nbdev_export) in the latest versions of nbdev.

Reply:
@butchtm yeah this was addressed in #242